admin/class-social-admin.php

```diff
@@ -89,32 +89,28 @@ public function get_meta_section() {
 		wp_nonce_field( 'yoast_free_metabox_social', 'yoast_free_metabox_social_nonce' );
 
 		if ( $opengraph === true ) {
-			$tabs[] = new WPSEO_Metabox_Form_Tab(
+			$tabs[] = new WPSEO_Metabox_Collapsible(
 				'facebook',
 				$this->get_social_tab_content( 'opengraph', $social_meta_fields ),
-				'<span class="screen-reader-text">' . __( 'Facebook / Open Graph metadata', 'wordpress-seo' ) . '</span><span class="dashicons dashicons-facebook-alt"></span>',
+				__( 'Facebook', 'wordpress-seo' ),
```

Comment:
I can't find where it is specified the ` / Open Graph metadata` should be removed.

Reply:
This was decided in internal communication.